src/__tests__/bad/js/internal-modules.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import picka from '@pika/pack/dist-src/index';
```

Comment:
Wouldn't this also fail because of a missing dependency? Can we require an internal module of a dependency we have?

Reply:
Thanks for the correction, this test case should be about `import/no-unresolved`. I added another test case for `import/no-internal-modules`